docs/user/docs/index.md

```diff
@@ -124,9 +98,6 @@ There are 3 roles you can have in a project context:
 - **Model manager** - can do same as both users above but also can invite or
   remove users from the managed projects and control their access rights.
 
-You can find more details about
-[how to request a session here](sessions/request.md).
-
```

Comment:
is this still linked somewhere?

Reply:
There a few broken links, but I fixed them all.